examples/resources/spectrocloud_application_profile/resource.tf

```diff
@@ -1,37 +1,67 @@
+  }
+  # Sample Helm Tier
+  pack {
+    name            = "kafka-tier"
+    type            = "helm"
+    registry_uid    = "605f66e748cd5ef5bd070ba8"
+    source_app_tier = "638f21ac7f63fb9677625873"
+    manifest {
+      name    = "test"
+      content = "annotations:\n  category: Infrastructure\napiVersion: v2\nappVersion: 3.3.1\ndependencies:\n  - condition: zookeeper.enabled\n    name: zookeeper\n    repository: https://charts.bitnami.com/bitnami\n    version: 11.x.x\n  - name: common\n    repository: https://charts.bitnami.com/bitnami\n    tags:\n      - bitnami-common\n    version: 2.x.x\ndescription: Apache Kafka is a distributed streaming platform designed to build real-time pipelines and can be used as a message broker or as a replacement for a log aggregation solution for big data applications.\nengine: gotpl\nhome: https://github.com/bitnami/charts/tree/main/bitnami/kafka\nicon: https://bitnami.com/assets/stacks/kafka/img/kafka-stack-220x234.png\nkeywords:\n  - kafka\n  - zookeeper\n  - streaming\n  - producer\n  - consumer\nmaintainers:\n  - name: Bitnami\n    url: https://github.com/bitnami/charts\nname: kafka\nsources:\n  - https://github.com/bitnami/containers/tree/main/bitnami/kafka\n  - https://kafka.apache.org/\nversion: 20.0.0"
+    }
+  }
+  # Sample Manifest Tier
+  pack {
+    name          = "manifest-3"
+    type          = "manifest"
+    install_order = 0
+    manifest {
+      name    = "test-manifest-3"
+      content = "apiVersion: apps/v1\nkind: Deployment\nmetadata:\n  name: nginx-deployment\n  labels:\n    app: nginx\nspec:\n  replicas: 3\n  selector:\n    matchLabels:\n      app: nginx\n  template:\n    metadata:\n      labels:\n        app: nginx\n    spec:\n      containers:\n      - name: nginx\n        image: nginx:1.14.2\n        ports:\n        - containerPort: 80"
+    }
+  }
+  # Sample DB Tier's
+  pack {
```

Comment:
We can also take this section and use it as a DB example and add it as a section in the template file. That way, we also have a DB example heading 😄